### DISCUSSION TITLE
When will a RC come out for 3.12.0?

### DISCUSSION THREAD

**Discussion by jschabowsky** [rabbitmq/hop#246] [General]
I am just curious as to when the 3.12.0-RC1 will be available in https://packagecloud.io/rabbitmq/maven-milestones. If this is documented somewhere, just point me to it. Thanks in advance!

**Comment by acogoluegnes** [comment]
If you're referring to the release that contains the added methods for the consumers endpoint you contributed, this is 3.11.0, it's [already available](https://github.com/rabbitmq/hop/releases/tag/v3.11.0).
